definitions/npm/react-splitter-layout_v3.x.x/flow_v0.53.x-/react-splitter-layout_v3.x.x.js

```diff
@@ -0,0 +1,25 @@
+declare type $$reactsplitterlayout$$ReactElement =
+  | boolean
+  | string
+  | number
+  | React$Element<any>;
```

Comment:
wait, I need null and undefined there too. But I can't make it work properly at the moment.

Reply:
I decided to use React$Node instead, because I couldn't make it accept void or undefined in the array, Flow kept giving me strange errors...